.github/workflows/go.yml

```diff
@@ -3,32 +3,29 @@ on: [push]
 jobs:
 
   build:
-    name: Build
-    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+          go-version: [1.14.x, 1.15.x, 1.16.x, 1.17.x]
```

Comment:
I don't think we need to test all versions with GHA. Just the one we are using for this provider. It's because there's no client/other package depending on this

Reply:
OK

Reply:
I will keep this `[1.16.x, 1.17.x]`